shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -285,6 +297,33 @@ protected void fetchColumns(final CommandLine cl, final ScannerBase scanner,
     }
   }
 
+  private void fetchColumsWithCFAndCQ(CommandLine cl, Scanner scanner, ScanInterpreter interpeter) {
+    String cf = "";
+    String cq = "";
+    if (cl.hasOption(scanOptCfOptions.getOpt())) {
+      cf = cl.getOptionValue(scanOptCfOptions.getOpt());
+    }
+    if (cl.hasOption(scanOptColQualifier.getOpt())) {
+      cq = cl.getOptionValue(scanOptColQualifier.getOpt());
+    }
+
+    if (cf.isEmpty() && !cq.isEmpty()) {
+      String formattedString = String.format(
+          "Option - %s when used with (- %s" + "cannot be empty ", scanOptCfOptions.getOpt(),
```

Comment:
Like above, the spacing is also a little off here. This is how it looks in the shell: ` Option - cf when used with (- cqcannot be empty`